root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -431,21 +431,31 @@ class Dialog {
     });
   }
 
+  isUntypedSeries() {
+    return this.source.entityType === 'series' &&
+           this.source.typeID() === '\xA0';
```

Comment:
Where does this value come from?

Reply:
From https://github.com/metabrainz/musicbrainz-server/blob/89d33473dadc93cb165ba8bece1c08117ccb9c30/lib/MusicBrainz/Server/Plugin/FormRenderer.pm#L203

Reply:
A more robust check might be something like `&& !isPositiveInteger(this.source.typeID())` in case the empty placeholder changes.